### DISCUSSION TITLE
How to get site points of RVD on the surface ?

### DISCUSSION THREAD

**Discussion by liu411417492** [BrunoLevy/geogram#179] [Q&A]
Hi, thanks again for this brilliant library。
I have a question here, how can i get the site points of RVD on the surface ?
i can get every convex cell use for_each_polyhedron(),but can i project them on the polygon which the on the surface? is there any api to do this?

**Comment by BrunoLevy** [comment]
(I'm sorry I am a bit slow to answer, need to finish a batch of changes / bugfixes then I'll come back to you right after).
